friendsofsymfony/rest-bundle/2.2/etc/packages/fos_rest.yaml

```diff
@@ -0,0 +1,17 @@
+# Read the documentation: https://symfony.com/doc/master/bundles/FOSRestBundle/index.html
+fos_rest: ~
+# Uncomment to activate features
```

Comment:
@fabpot want to tell you everything that is in comment is not worth it to be present?